.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+  directory: "/"
+  schedule:
+    interval: "weekly"
+    ignore:
+        # These are peer deps of Cargo and should not be automatically bumped
+        - dependency-name: "iced"
+        #- dependency-name: "crates-io"
+    rebase-strategy: "enabled"
```

Comment:
You can remove this, as it is enabled by default.

Reply:
ok